static/app/views/issueList/searchBar.tsx

```diff
@@ -23,18 +24,22 @@ import usePageFilters from 'sentry/utils/usePageFilters';
 import type {WithIssueTagsProps} from 'sentry/utils/withIssueTags';
 import withIssueTags from 'sentry/utils/withIssueTags';
 
-const getSupportedTags = (supportedTags: TagCollection) =>
-  Object.fromEntries(
-    Object.keys(supportedTags).map(key => [
-      key,
-      {
-        ...supportedTags[key],
-        kind:
-          getFieldDefinition(key)?.kind ??
-          (supportedTags[key].predefined ? FieldKind.FIELD : FieldKind.TAG),
-      },
-    ])
+const getSupportedTags = (supportedTags: TagCollection, org: Organization) => {
+  const include_priority = org.features.includes('issue-priority-ui');
+  return Object.fromEntries(
+    Object.keys(supportedTags)
+      .map(key => [
+        key,
+        {
+          ...supportedTags[key],
+          kind:
+            getFieldDefinition(key)?.kind ??
+            (supportedTags[key].predefined ? FieldKind.FIELD : FieldKind.TAG),
+        },
+      ])
+      .filter(([key, _]) => (key === FieldKey.ISSUE_PRIORITY ? include_priority : true))
```

Comment:
I also don't think it's necessary to filter here - the modification you added in `tagStore.tsx` should have already filtered it out from `tags`

Reply:
Removing this ends up showing `issue.priority` in the search

Reply:
Hmm I must be missing something. Looks good then!